### PR TITLE
OAuth token timeout tests

### PR DESCRIPTION
**Pull request by gsleeman**
Tests for the presence of timeout values in the global oauth token config, and tests whether inactivity timeouts are working for oauthaccesstokens

Some of these tests are expected to fail until https://github.com/openshift/managed-cluster-config/pull/766 is in prod

### CODE REVIEW

**Review by whereswaldon — CHANGES_REQUESTED** [review]
This looks reasonable to me, though we do need to switch to a thread-safe viper.

Have you tested this locally?

File: pkg/e2e/verify/oauth_tokens.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+	"net/url"
+	"time"
+
+	"github.com/Masterminds/semver"
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	userv1 "github.com/openshift/api/user/v1"
+	"github.com/spf13/viper"
```
[the remark this answers is not in the document]
Comment (gsleeman) [reply pkg/e2e/verify/oauth_tokens.go]: Thanks for that.

I've tested locally with a kubeconfig and focus test. Do I need to do a full test run as well?

[the remark this answers is not in the document]
Comment (whereswaldon) [reply pkg/e2e/verify/oauth_tokens.go]: No, the focus test is sufficient.

**Review by whereswaldon — APPROVED** [review]

**Pull request merged**